### DISCUSSION TITLE
Crackling popping lagging, impossible to use Audiogridder

### DISCUSSION THREAD

**Discussion by karmamelle** [apohl79/audiogridder#1261] [Help]
Hi there! I recently installed Audiogridder on my mac and I am trying to use it with crossover, but unluckily every time I open the plugin I start to get clicks and pops and the sound is glitchy, so I tried to change the buffer size and when I do so it crashes...
Some one here can help me?
Thanks

**Comment by williamcorney** [comment]
I use audiogridder with a VST on Ventura 13.4 64bit and Logic Pro  .  I can’t vouch for all plugins and DAWs but can say it does work for me with this combination

Have you tried wineasio?  Do you need to be able to record, or you just want latency free audio of a windows VST?

**Reply by karmamelle** [reply]
Thanks for answering William.. I don't understand what you are saying, I am not an expert in programming.. my set up on my mac studio max is: Bitwig 5 as master daw running normally, Crossover wine etc. with Bitwig windows version installed, which works perfectly infact I can open many plugins from there... then I discovered Audiogridder, so I installed the plugin on my Bitwig mac version and the server on my virtual Windows.. right now this configuration is not working because every time I open a vst inside Audiogridder the audio goes away, if I change the buffer size I get very bad cracks and pops.. If I keep change the buffer size Audiogridder crashes... maybe there is something I should do or maybe is simply not done to be used with a virtual machine? I hope some one can help me.
Thanks

**Comment by williamcorney** [comment]
I am just an amateur too just fumbling out till I know enough to get something to work.  Just read lots of posts, keep soaking up information until the myst starts clearing.  I can't say I understand all the steps of the processes I do.  I understand enough to know they need to be done and a vague idea of what each step is doing.  I have achieved getting VST working with audiogridder and wineasio so I know what to look for now which makes it a whole lot easier.  I had to battle through the issue alone with very little step by step instructions as many posts out there are for people with linux instead.

I have made a tutorial of my journey with wineasio in a repo called wineasio which you'll find if you look under my name.  There's screenshots there too

Does your setup include a digital keyboard or another instrument ?

You might have to adapt these instructions if you are using on an M1/M2 mac.  I don't have access to or experience with a real M1/M2 but from what I have read online it sounds like still possible.  It sounds like everything is done with Rosetta which I presume you know about.

I don't know what you did or didn't understand about my message so ill expand on all points

DAW is Desktop Audio Workstation ( its the industry word for software like Bit Wig 5 or in my case Logic Pro ).

Latency issues mentioned are that audio works, but there's an annoying delay in the audio from when you press a key on keyboard to when you hear the generated sound.  Sometimes this is accompanied by poor audio quality or pops and crackles.

It sounds like it might not be worth your while - if everything is working perfectly with just crossover and bigwig installed within the same bottle as the VSTS.  I presume you know what I mean by a bottle but in case - it's just a container for a wine prefix.  Wine prefixes are just like self contained instances of wine.

Two solutions to these issues in my experience which I have found worked are 

(1)  if you need to record the audio as well as address the quality issues - using audiogridder in a wine prefix.  I get no latency issues or quality problems doing this
(2)  if you just want to play and instrument ( without recording) then you can look at a plugin called wineasio and a piece of software called JACK.   In your scenario - jack is installed on your Mac.  wine asio is a dll installed inside the wine prefix which gives you a virtual audio interface called wineasio - which basically routes the audio from the VST to the host computer in the most efficient way possible ( which resolves any issues such as latency or audio quality).

**Reply by karmamelle** [reply]
Hi William, thanks for answering. I did not understand the last part of your post, now it is clear what you are asking.. anyway I would like to use the vst to be able to record it and  automate it.. as I am already using windows vsts in windows DAW inside the bottle and I can send midi from my main DAW and get back the audio, I would use Audiogridder mainly for avoid having two DAWs opened in the same time and to be able to automate parameters and in general make things more easy. Some latency it is fine unless it makes impossible to listen to the sounds on time.. cracks and pops of course are unacceptable.
